src/main/java/com/atlauncher/data/Instance.java

```diff
@@ -115,12 +116,7 @@
 import com.atlauncher.mclauncher.MCLauncher;
 import com.atlauncher.network.Analytics;
 import com.atlauncher.network.DownloadPool;
-import com.atlauncher.utils.ArchiveUtils;
-import com.atlauncher.utils.ComboItem;
-import com.atlauncher.utils.FileUtils;
-import com.atlauncher.utils.OS;
-import com.atlauncher.utils.Utils;
-import com.atlauncher.utils.ZipNameMapper;
+import com.atlauncher.utils.*;
```

Comment:
Can you undo this, the preference is to keep individual imports rather than using star imports.